src/app/(proper_react)/redesign/(public)/LandingView.test.tsx

```diff
@@ -21,10 +21,15 @@ import Meta, {
   LandingNonUsDe,
   LandingNonUsFr,
   LandingUs,
+  LandingUsScanLimit,
 } from "./LandingView.stories";
 
 jest.mock("next-auth/react");
 jest.mock("../../../hooks/useTelemetry");
+jest.mock("next/navigation", () => ({
+  useRouter: jest.fn(),
+  usePathname: jest.fn(),
+}));
```

Comment:
Since you didn't add the `useRouter` and `usePathname` hooks in the code-under-test, I don't think you need these.

```suggestion
```